.github/workflows/test.yaml

```diff
@@ -46,6 +53,41 @@ jobs:
         run: 'echo "Lava status: ${{ steps.lava.outputs.status }}"'
       - name: Print report
         run: 'cat "${{ steps.lava.outputs.report }}"'
+      - name: Report unexpected config file
+        if: ${{ !endsWith(steps.lava.outputs.config, env.WANT_CONFIG) }}
+        run: |
+          echo "::error::unexpected config file: got: ${{ steps.lava.outputs.config }}, want: ${{ env.WANT_CONFIG }}"
+          exit 1
+      - name: Report unexpected status
+        if: ${{ steps.lava.outputs.status < env.WANT_MIN_STATUS || steps.lava.outputs.status > env.WANT_MAX_STATUS }}
+        run: |
+          echo "::error::unexpected status code: got: ${{ steps.lava.outputs.status }}, want: [${{ env.WANT_MIN_STATUS }}, ${{ env.WANT_MAX_STATUS }}]"
+          exit 1
+  test-default-lava-config:
+    env:
+      WANT_MIN_STATUS: 100 # ExitCodeInfo
+      WANT_MAX_STATUS: 104 # ExitCodeCritical
+      WANT_CONFIG: lava.yaml
+    name: Test default lava config
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Run Lava Action
+        id: lava
+        uses: ./
+        env:
+          TEST_WORKDIR: testdata
+        continue-on-error: true
+      - name: Print status
+        run: 'echo "Lava status: ${{ steps.lava.outputs.status }}"'
+      - name: Print report
+        run: 'cat "${{ steps.lava.outputs.report }}"'
+      - name: Report unexpected config file
+        if: ${{ !endsWith(steps.lava.outputs.config, env.WANT_CONFIG) }}
+        run: |
+          echo "::error::unexpected config file: got: ${{ steps.lava.outputs.config }}, want: ${{ env.WANT_CONFIG }}"
+          exit 1
       - name: Report unexpected status
         if: ${{ steps.lava.outputs.status < env.WANT_MIN_STATUS || steps.lava.outputs.status > env.WANT_MAX_STATUS }}
```

Comment:
Given that you are using `lava.yaml` here, you don't need to check a range. See `test` job.